src/Components/PlaylistThumbnail/playlistThumbnail.css

```diff
@@ -0,0 +1,40 @@
+.PlaylistThumbnail {
+    width: 20rem;
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+.thumbnail-container h2,
+.thumbnail-container p {
+    margin: 0;
+    padding: 0.5rem;
+}
+
+.thumbnail-container h2 {
+    cursor: pointer;
+}
+
+.thumbnail-container {
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem;
+    border: 1px solid var(--binge-red);
+}
+
+.thumbnail-left {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    text-align: left;
+}
+
+.thumbnail-left>* {
+    text-decoration: none;
+    color: whitesmoke;
+}
+
+.thumbnail-right>* {
```

Comment:
spacing can make things a lot more reasdable

Reply:
Noted, this is VS Code's default formatter doing it's thing.